src/platform/site-wide/user-nav/tests/selectors.unit.spec.js

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import { set } from 'lodash/fp';
+import set from '../../../utilities/data/set';
```

Comment:
I think we can do 
```js
import set from 'platform/utilities/data/set';
```
right?

Reply:
Yeah, the relative path can be used here. I added it this way to match the other imports in the file, but I can always update it if you want.

Reply:
I went ahead and updated the import ✅. I also updated the `localStorage` import, since I was already in the file.